src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,56 @@
+import cn from 'classnames';
+import { useSearchParams } from 'react-router-dom';
+import { SearchLink } from './SearchLink';
+
 export const PeopleFilters = () => {
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const sex = searchParams.get('sex') || '';
+  const centuries = searchParams.getAll('centuries') || [];
+
+  const arrayOfCenturies: string[] = ['16', '17', '18', '19', '20'];
+
+  function handleQueryChange(event: React.ChangeEvent<HTMLInputElement>) {
+    const params = new URLSearchParams(searchParams);
+
+    params.set('query', event.target.value);
+
+    if (event.target.value.length === 0) {
+      params.delete('query');
+    }
+
+    setSearchParams(params);
+  }
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        <SearchLink
+          className={cn({
+            'is-active': sex === '',
+          })}
+          params={{ sex: null }}
+        >
+          All
+        </SearchLink>
+        <SearchLink
+          className={cn({
+            'is-active': sex === 'm',
+          })}
+          params={{ sex: 'm' }}
+        >
+          Male
+        </SearchLink>
+        <SearchLink
+          className={cn({
+            'is-active': sex === 'f',
+          })}
+          params={{ sex: 'f' }}
+        >
+          Female
+        </SearchLink>
```

Comment:
nothing serious but you can use the map method here

Reply:
Implement